lib/model/content.dart

```diff
@@ -1072,6 +1072,19 @@ class _ZulipInlineContentParser {
       return GlobalTimeNode(datetime: datetime, debugHtmlNode: debugHtmlNode);
     }
 
+    if (localName == 'audio' && className.isEmpty) {
+      final srcAttr = element.attributes['src'];
+      if (srcAttr == null) return unimplemented();
+
+      final titleAttr = element.attributes['title'];
+
+      final link = LinkNode(
+        url: srcAttr,
+        nodes: [TextNode(titleAttr ?? srcAttr)]);
```

Comment:
It feels a bit messy to be transforming this into a plain link, the same thing that an actual link would have been parsed as. Cleaner would be to give it its own node type, and handle this transform at build time.

But this is fine for getting something quickly that we can ship. The cleaner thing probably requires a bit of refactoring to make it cooperate with LinkNode and that infrastructure.